src/apirequest.ts

```diff
@@ -160,7 +160,11 @@ async function createAPIRequestAsync<T>(parameters: APIRequestParams) {
 
   // Parse urls
   if (options.url) {
-    options.url = urlTemplate.parse(options.url).expand(params);
+    let url = options.url;
```

Comment:
This is an unrelated fix: without this, the existing code does not compile on the new TypeScript version.